gdpr/utils.py

```diff
@@ -56,7 +56,10 @@ def get_reversion_versions(obj: Any) -> QuerySet:
     if hasattr(Version.objects, 'get_for_object'):
         return Version.objects.get_for_object(obj).order_by('id')
     content_type = ContentType.objects.get_for_model(obj.__class__)
-    return Version.objects.filter(content_type=content_type, object_id=obj.pk).order_by('id')
+    if isinstance(obj.pk, int):
+        return Version.objects.filter(content_type=content_type, object_id_int=obj.pk).order_by('id')
+    else:
+        return Version.objects.filter(content_type=content_type, object_id=obj.pk).order_by('id')
```

Comment:
this is little bit tricky. Integer primary keys are stored to the object_id_int field which has db index

Reply:
I'm just thinking if there shouldn't be some check that the `object_id_int` exists. But the tests are passing so I'm guessing there is no need for that.

Reply:
it is not necessary, because the old version have it and the new version of reversion have method get_for_object